Netwrix/netwrix_auditor/ingest/parser.yml

```
@@ -0,0 +1,87 @@
+name: netwrix_auditor_parser
+pipeline:
+  - name: cef_parser
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: general-setter
+    description: ''
+    filter: ''
+    external: null
+  - name: user-email-setter
+    description: ''
+    filter: '{{ "@" in cef_parser.message.suser }}'
+    external: null
+  - name: user-name-domain-setter
+    description: ''
+    filter: '{{ "\\" in cef_parser.message.suser }}'
+    external: null
+  - name: user-only-setter
+    description: ''
+    filter: '{{ "\\" not in cef_parser.message.suser and "@" not in cef_parser.message.suser }}'
+  - name: document-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.cat == "Document" }}'
+    external: null
+  - name: logon-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Logon Activity" }}'
+    external: null
+  - name: exchange-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Exchange Online" }}'
+    external: null
+stages:
+  general-setter:
+    actions:
+      - set:
+          event.kind: '{{ cef_parser.message.DeviceProduct }}'
+          event.category: '{{ cef_parser.message.DeviceEventClassID |lower }}'
+          event.severity: '{{ cef_parser.message.Severity }}'
+          event.start: '{{ cef_parser.message.start }}'
+        filter: ''
+        name: set
+  document-event-setter:
+    actions:
+      - set:
+          file.path: '{{ cef_parser.message.filePath.split("/")[3:] |join("/") }}'
+          event.action: '{{ cef_parser.message.Name | lower }}'
+        filter: ''
+        name: set
+  logon-event-setter:
+    actions:
+      - set:
+          event.reason: '{{ cef_parser.message.msg }}'
+          server.address: '{{ cef_parser.message.shost }}'
```

To preserve the semantics of `shost`, I suggest using `source.domain`:
```suggestion
          source.domain: '{{ cef_parser.message.shost }}'
```